prime-router/src/testIntegration/resources/datatests/mappinginventory/catchall/xtn/XTN-to-ContactPoint.csv

```diff
@@ -1,24 +0,0 @@
-HL7 v2,,,,,,Condition (IF True),,,HL7 FHIR,,,,,,,,,Comments
```

Comment:
The test file for this and the other removed XTN test files was removed in [this PR](https://github.com/CDCgov/prime-reportstream/pull/13119), so this is just removing files that are not invoked anywhere.